### ISSUE TITLE
read_ncswan issue for opening SWAN output files

### ISSUE THREAD

**Issue by DiegoBindoni** [wavespectra/wavespectra#126]
Good morning,
i have SWAN output files coming from COAWST coupled simulations, saved in netCDF file using SPECOUT over a FRAME selected. I would like to plot the spectra in a specific point, but when i try to use read_ncswan, i obtain always this error.

```
File "/tmp/ipykernel_241878/1539093376.py", line 4, in <module>
    dset = read_ncswan(swan_spectra)

  File "/data/nobackup/bindoni/miniconda3/envs/wrfpython37/lib/python3.7/site-packages/wavespectra/input/ncswan.py", line 50, in read_ncswan
    return from_ncswan(dset)

  File "/data/nobackup/bindoni/miniconda3/envs/wrfpython37/lib/python3.7/site-packages/wavespectra/input/ncswan.py", line 63, in from_ncswan
    dset = dset.rename(MAPPING)

  File "/data/nobackup/bindoni/miniconda3/envs/wrfpython37/lib/python3.7/site-packages/xarray/core/dataset.py", line 3389, in rename
    f"cannot rename {k!r} because it is not a "

ValueError: cannot rename 'direction' because it is not a variable or dimension in this dataset
```

Could you please help me figure out what's the problem?

**Comment by rafa-guedes** [comment]
@DiegoBindoni thanks for reporting this - it is a bug and it has been fixed in https://github.com/wavespectra/wavespectra/pull/106. We should hopefully be merging and releasing shortly, in the meanwhile if you need this more urgently you could clone the code, checkout the `oceanum` branch and installing it locally.

**Comment by rafa-guedes** [comment]
Fixed in v4.0.0
